File: CS50x/PSET6/sentimental-mario-less/mario.py
# // Building a right-aligned pyramid

from cs50 import get_int

# Get user input of the height
while True:
    n = get_int("Height: ")
    if n > 0 and n < 9:
        break

# Rows of the pyramid
for i in range(n):
    # Columns of the pyramid
    for j in range(n):
        # Print blankspaces and hashes
        if i + j < n - 1:
            print(" ", end="")
        else:
            print("#", end="")
    print()
    

File: CS50x_/PSET1/mario-less/mario.c
// Building a right-aligned pyramid
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Get user input of the height
    int n;
    do
    {
        n = get_int("Height: ");
    }
    while (n < 1 || n > 8);

    // Rows of the pyramid
    for (int i = 0; i < n; i++)
    {
        // Columns of the pyramid
        for (int j = 0; j < n; j++)
        {
            // Print blankspaces and hashes
            if (i + j < n - 1)
            {
                printf(" ");
            }
            else
            {
                printf("#");
            }
        }

        printf("\n");
    }
}